### PR TITLE
feature/open-ide

### LINKED ISSUE #111: Open with IDE of choice
Would it possible to have an option to open a specific project straight into the IDE of choice?

### PR DESCRIPTION
**Pull request by aguilaair**
Closes  #111 

This PR implements the ability to launch your Flutter projects directly into your IDE. Currently supports XCode and VSCode, but new IDEs may be added in the future by adding them to `supportedIDEs` in `settings.utils.dart`.

### CODE REVIEW

File: lib/src/modules/common/utils/open_link.dart [hunk 1]
```diff
@@ -17,19 +17,19 @@ Future<void> openLink(BuildContext context, String url) async {
 
 Future<void> openPath(BuildContext context, String url) async {
   if (Platform.isWindows) {
-    await Process.start(
```
Comment (leoafarias) [inline lib/src/modules/common/utils/open_link.dart:20]: We probably should remove these strings from translation.

Comment (aguilaair) [reply lib/src/modules/common/utils/open_link.dart:20]: Fixed in main

**Comment by gauravmehta13** [comment]
This will be very helpful, any eta for this to come in release?

**Comment by aguilaair** [comment]
@gauravmehta13 Should be ready soon, and I think the release will follow soon after as we haven't done a release in forever 😄

**Review by leoafarias — APPROVED** [review]
LGTM

**Comment by gauravmehta13** [comment]
> @gauravmehta13 Should be ready soon, and I think the release will follow soon after as we haven't done a release in forever 😄

Thanks

**Pull request merged**